file/file_prefetch_buffer.cc

```diff
@@ -312,11 +320,8 @@ Status FilePrefetchBuffer::PrefetchAsync(const IOOptions& opts,
     // swap the buffers.
     curr_ = curr_ ^ 1;
     // Update prefetch_size as length has been updated in CopyDataToBuffer.
-    prefetch_size = length + readahead_size;
```

Comment:
Why is this not required anymore? The length could have changed due to the calls to CopyDataToBuffer().

Reply:
Yes it was my mistake. By mistake, I removed this and next line 319.